internal/endpoint/sqs.go

```diff
@@ -65,37 +65,30 @@ func (conn *SQSConn) Send(msg string) error {
 	conn.t = time.Now()
 
 	if conn.svc == nil && conn.session == nil {
+		var creds *credentials.Credentials
 		credPath := conn.ep.SQS.CredPath
-		credProfile := conn.ep.SQS.CredProfile
-		var sess *session.Session
-		if credPath != "" && credProfile != "" {
-			sess = session.Must(session.NewSession(&aws.Config{
-				Region:      aws.String(conn.ep.SQS.Region),
-				Credentials: credentials.NewSharedCredentials(credPath, credProfile),
-				MaxRetries:  aws.Int(5),
-			}))
-		} else if credPath != "" {
-			sess = session.Must(session.NewSession(&aws.Config{
-				Region:      aws.String(conn.ep.SQS.Region),
-				Credentials: credentials.NewSharedCredentials(credPath, "default"),
-				MaxRetries:  aws.Int(5),
-			}))
-		} else {
-			sess = session.Must(session.NewSession(&aws.Config{
-				Region:     aws.String(conn.ep.SQS.Region),
-				MaxRetries: aws.Int(5),
-			}))
+		if credPath != "" {
+			credProfile := conn.ep.SQS.CredProfile
+			if credProfile == "" {
+				credProfile = "default"
+			}
+			creds = credentials.NewSharedCredentials(credPath, credProfile)
 		}
-		// Create a SQS service client.
+		sess := session.Must(session.NewSession(&aws.Config{
+			Region:      aws.String(conn.ep.SQS.Region),
```

Comment:
I think the `region` should be as well added to the documentation at https://github.com/tile38/tile38.github.io/blob/master/src/website/root/commands/sethook.md#L141. Would it eventually make sense to set a default of `us-east-1`?

Reply:
I agree. I'll make sure the docs are more descriptive right after a merge the PR.
I'm not sure about defaulting to us-east-1 though. Mainly because the URL is already a little strange `sqs://us-east-1:928374716/tile38`. 
But if so, do you think something like this? `sqs://928374716/tile38`

Reply:
I think the most straight foward way would be to use the queue URL directly, such as `https://sqs.eu-central-1.amazonaws.com/123456789/myqueue`. This string seems to easily be parseable. 

If that doesn't seem favorable, I'd at least recommend to change the `myqueueid` portion in the docs to `myaccountnumber` to make it more clear.

Reply:
Ok. I just pushed an update that allows for both:
```
https://sqs.eu-central-1.amazonaws.com/123456789/myqueue
sqs://eu-central-1:123456789/myqueue
```

I'll probably update the docs to support the first option as primary since it's what AWS suggests.